README.md

```diff
@@ -8,277 +8,36 @@
 
-## Getting Started
+## First time setup
+1. Make sure you have [`Node.js`](https://nodejs.org/en/) version >= 18 installed
+2. Run [script to patch your `/etc/hosts`](https://github.com/RedHatInsights/insights-proxy/blob/master/scripts/patch-etc-hosts.sh)
+3. Make sure you are using [Red Hat proxy](http://hdn.corp.redhat.com/proxy.pac)
 
-The Patchman UI is a sub application under the Red Hat insights platform with [Insights chroming app as a wrapper](https://github.com/RedHatInsights/insights-chrome).
-
-There are 2 technologies used for integration with backend and deployment purposes: 
-1. [Insights Proxy](https://github.com/RedHatInsights/insights-proxy)
-2. [Webpackproxy](https://github.com/RedHatInsights/frontend-components/tree/master/packages/config#useproxy)
-
-Note: the difference is insights proxy has to be reinitialised everytime you want to develop, while webpack proxy is auto initialized when you  run the command ```npm run start:proxy:beta```.
-
-There is [a comprehensive quick start guide in the Storybook Documentation](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/welcome/quickStart/DOC.md) to setting up an Insights environment complete with:
-
-- [Insights Frontend Starter App](https://github.com/RedHatInsights/frontend-starter-app)
-
-- [Insights Chroming](https://github.com/RedHatInsights/insights-chrome)
-
-- [Insights Proxy](https://github.com/RedHatInsights/insights-proxy)
-
-Note: You will need to set up the Insights environment if you want to develop with the starter app due to the consumption of the chroming service as well as setting up your global/app navigation through the API.
-
-## Run the app for the first time
-
-1. ```git clone git@github.com:RedHatInsights/patchman-ui.git```
-    - clone the Patch UI into your machine
-
-2. ```npm install```
-
-3. ```npm run start:proxy```
-    - use this command if you want to develop Patch UI using insights proxy. The command starts webpack bundler and serves the files with webpack dev server
-    - While you run the command above, you will be prompted to choose an enviroment.
-
-Open browser with the URL generated by the webpack dev server your terminal
-
-### Testing
-
-- `npm run verify` will run linters and tests
-- Travis is used to test the build for this code.
-  - You are always notified on failed builds
-  - You are only notified on successful builds if the build before it failed
-  - By default, both `push` events as well as `pull_request` events send notifications
-  - Travis is defaulted to notify #insights-bots
-  - `npm run test` will run only Jest tests
-  - `npm run test:update` is used to update the test snapshots
+## Running locally
+1. Make sure you are connected to the Red Hat VPN
+2. Install dependencies with `npm install`
+3. Run development server with `npm run start:proxy` and select desired environment (`stage-preview` is recommended)
```

Comment:
I would also mention that one needs to have podman virtual machine started since the chrome is served as a container.

Reply:
@gkarat but that's only if you are running without proxy, or?

Reply:
@leSamo, chrome assets are served from a container in all environments irregardless proxy/non-proxy.

Reply:
I'm trying to keep the readme as simple as possible without any special quirks and configuration. Do you need to run the container even if you are running the app regularly? :thinking: